geth/api/api.go

```diff
@@ -188,3 +193,35 @@ func (api *StatusAPI) JailCall(chatID, this, args string) string {
 func (api *StatusAPI) JailBaseJS(js string) {
 	api.b.jailManager.BaseJS(js)
 }
+
+// TODO: API package this stuff
+func (api *StatusAPI) Notify(token string) string {
+	log.Info("NOTIFY LOG", "token", token)
+
+	var NP fcm.NotificationPayload
```

Comment:
Missed import I suppose: `import "github.com/NaySoftware/go-fcm/fcm.go"` at the top

Reply:
Thanks